sdk/sysvar/src/slot_hashes.rs

```diff
@@ -96,21 +97,23 @@ pub struct PodSlotHashes {
     slot_hashes_end: usize,
 }
 
+#[cfg(feature = "bytemuck")]
 impl PodSlotHashes {
+    #[cfg(feature = "bincode")]
```

Comment:
I'll do some follow-up work to remove the requirement on `bincode` for `get_sysvar` -- thank you for fighting through this